tools/src/launcher/Main.hs

```diff
@@ -515,11 +523,18 @@ spawnNode (path, args, mbLogPath) = do
             logInfo "Node has started"
             return (ph, asc)
 
-runWallet :: (FilePath, [Text]) -> M ExitCode
-runWallet (path, args) = do
-    logNotice "Starting the wallet"
-    view _1 <$>
-        liftIO (readProcessWithExitCode path (map toString args) mempty)
+runWallet :: Bool -> (FilePath, [Text]) -> M ExitCode
+runWallet shouldLog (path, args) = do
+    putText "Starting the wallet"
+    if shouldLog then
+        liftIO $ do
+        (_, stdO, stdE, pid) <- runInteractiveProcess path (map toString args) Nothing Nothing
+        withAsync (forever $ IO.hGetLine stdO >>= IO.hPutStrLn stdout . ("[wallet] " <>)) $ \_ ->
```

Comment:
I know this was a problematic part, but:
1. Looks hacky. Have you considered using `turtle`?
2. Don't you want to at least (maybe) `cancel` these two async threads?

Reply:
withAsync does the canceling

Reply:
Can someone just give this a go ? On my machine I get this error
```
+ rm -v './run//topology*.yaml'
rm: ./run//topology*.yaml: No such file or directory
```

Reply:
Also I think this is related to https://github.com/input-output-hk/cardano-sl/issues/1869